addons/website_mass_mailing/controllers/main.py

```diff
@@ -46,35 +45,7 @@ def subscribe(self, list_id, email, **post):
             subscription.opt_out = False
         # add email to session
         request.session['mass_mailing_email'] = email
-        mass_mailing_list = request.env['mailing.list'].sudo().browse(list_id)
         return {
             'toast_type': 'success',
-            'toast_content': mass_mailing_list.toast_content
+            'toast_content': _("Thanks for subscribing!"),
```

Comment:
Kinda a regression?

Reply:
I guess but is not it easy to keep as editable content in the popup itself? (kinda like you did for the editable message after countdown)

Reply:
I don't think it would add much value to the product IMHO.

I don't think people actually customized it (you have to go to the mailing list in the backend).

I'd just KISS, less code, less UI, more clear.